src/concrete/ml/common/utils.py

```diff
@@ -605,3 +606,50 @@ def array_allclose_and_same_shape(
     assert isinstance(b, numpy.ndarray)
 
     return a.shape == b.shape and numpy.allclose(a, b, rtol, atol, equal_nan)
+
+
+def check_rounding_threshold(rounding_threshold_bits):
```

Comment:
did not realize that this was not only checking but also "building" the object. I would rename it to something else then, like `process_rounding_threshold_bits`

Reply:
Sure!